### PR TITLE
EeroQ notebook updates

### PR DESCRIPTION
**Pull request by bharat-thotakura**
Cleans up the EeroQ optimization notebooks to be closer in standardization to the other hardware optimization notebooks 

(additionally contains very minor nits for the QSCOUT notebook discovered via editing the EeroQ notebooks)

### CODE REVIEW

File: docs/source/optimizations/eeroq/eeroq_css.ipynb [hunk 1]
```diff
@@ -18,18 +18,20 @@
   },
```
Comment (pgoiporia) [inline docs/source/optimizations/eeroq/eeroq_css.ipynb:18]: <p>This probably doesn't need to be a subheading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: docs/source/optimizations/eeroq/eeroq_qss.ipynb [hunk 2]
```diff
@@ -18,18 +18,20 @@
   },
```
Comment (pgoiporia) [inline docs/source/optimizations/eeroq/eeroq_qss.ipynb:18]: <p>Same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Review by pgoiporia — APPROVED** [review]
LGTM once you make those tiny tweaks. Thanks for making these changes, the notebook looks much better!

File: docs/source/optimizations/eeroq/eeroq_css.ipynb (hunk 1 above)
Comment (bharat-thotakura) [reply docs/source/optimizations/eeroq/eeroq_css.ipynb:18]: Yeah, it looked weird to me too in ReviewNB, but it's not set as a subheading afaict: 
image

Will correct in a follow-up PR if its wrong on Read-the-docs too

**Pull request merged**